mapper/orm-coordination-database-polling/src/main/avro/schema.avsc

```diff
@@ -0,0 +1,81 @@
+[
+   {
+      "namespace":"org.hibernate.search.mapper.orm.coordination.databasepolling.avro.generated.impl",
+      "type":"record",
+      "name":"DocumentRouteDescriptorDto",
+      "fields":[
+         {
+            "name":"routingKey",
+            "type":[
+               "string",
+               "null"
+            ]
+         }
+      ]
+   },
+   {
+      "namespace":"org.hibernate.search.mapper.orm.coordination.databasepolling.avro.generated.impl",
+      "type":"record",
+      "name":"DocumentRoutesDescriptorDto",
+      "fields":[
+         {
+            "name":"currentRoute",
+            "type":"org.hibernate.search.mapper.orm.coordination.databasepolling.avro.generated.impl.DocumentRouteDescriptorDto"
+         },
+         {
+            "name":"previousRoutes",
+            "type":[
+               {
+                  "type":"array",
+                  "items":"org.hibernate.search.mapper.orm.coordination.databasepolling.avro.generated.impl.DocumentRouteDescriptorDto"
+               },
+               "null"
```

Comment:
you're right the field cannot be null

Reply:
it seems that we cannot do that:
I got some error:
> Caused by: org.apache.avro.AvroRuntimeException: Field dirtyPaths type:ARRAY pos:2 does not accept null values

Reply:
I'm dropping the commit to fix it

Reply:
@fax4ever The message is dealing with `dirtyPaths`, which is another property, and I told you that I created a ticket because that one can be null.

What about `previousRoutes`, though? You did make that one non-nullable, didn't you?